packages/web-pkg/src/components/sidebar/SideBar.vue

```diff
@@ -338,6 +338,7 @@ export default defineComponent({
 
   &__body {
     overflow-y: auto;
+    overflow-x: hidden;
```

Comment:
What does this prevent?

Reply:
@pascalwengerter In case the email is long, the horizontal scroll appears. Happens to all emails at CERN
![Screenshot from 2022-05-13 16-28-42](https://user-images.githubusercontent.com/7430156/168306321-33ec670f-bcaf-4ee8-a6e9-f8329cdfdccb.png)